pyserini/encode/_slim.py

```diff
@@ -0,0 +1,62 @@
+import torch
+from transformers import AutoModelForMaskedLM, AutoTokenizer
+import numpy as np
+import scipy
+
+from pyserini.encode import QueryEncoder
+
+
+class SLIMQueryEncoder(QueryEncoder):
```

Comment:
I think to be consistent with naming conventions, this should be `SlimQueryEncoder`?